### PR TITLE
Fix/auth forms validation

### PR DESCRIPTION
**Pull request by johannesvedder**
- fix: no more issues with form validations at login and registration pages
- feat: added login and registration error messages

### CODE REVIEW

File: designer_v2/lib/features/auth/auth_form_controller.dart [hunk 1]
```diff
@@ -67,6 +68,7 @@ class AuthFormController extends StateNotifier<AsyncValue<void>> implements IFor
 
   // - Form controls
 
+  final FormControl<String> testControl = FormControl();
```
Comment (hig-dev) [inline designer_v2/lib/features/auth/auth_form_controller.dart:71]:
```suggestion
```

Remove this

**Review by hig-dev — CHANGES_REQUESTED** [review]
Looks good. One thing that I noticed was:

1. Go to register user
2. Fill password
3. Fill password confirmation
4. Accept terms

Now the register buttons gets activated although the email is empty.

**Comment by johannesvedder** [comment]
> Looks good. One thing that I noticed was:
> 
>     1. Go to register user
> 
>     2. Fill password
> 
>     3. Fill password confirmation
> 
>     4. Accept terms
> 
> 
> Now the register buttons gets activated although the email is empty.

Nice catch 👍 Should be fixed now.

**Review by hig-dev — APPROVED** [review]
Great

**Pull request merged**